Source/items.cpp

```diff
@@ -4886,6 +4886,67 @@ void PutItemRecord(int nSeed, uint16_t wCI, int nIndex)
 	}
 }
 
+#ifdef _DEBUG
+void DebugSpawnItem(std::string itemName, bool unique)
+{
+	if (ActiveItemCount >= MAXITEMS)
+		return;
+
+	bool onlygood = true;
+
+	int ii = AllocateItem();
+	Point pos = Players[MyPlayerId].position.tile;
+	GetSuperItemSpace(pos, ii);
+	std::transform(itemName.begin(), itemName.end(), itemName.begin(), [](unsigned char c) { return std::tolower(c); });
+
+	std::chrono::steady_clock::time_point begin = std::chrono::steady_clock::now();
+	MonsterStruct fake_m;
+	fake_m.MData = &MonsterData[0];
+	fake_m._uniqtype = 0;
+	for (int i = 0;; i++) {
+		std::chrono::steady_clock::time_point end = std::chrono::steady_clock::now();
+		int diff = std::chrono::duration_cast<std::chrono::milliseconds>(end - begin).count();
+		if (diff > 3000) {
+			NetSendCmdString(1 << MyPlayerId, "ITEM NOT FOUND IN 3 SECONDS!");

```

Comment:
We should make `DebugSpawnItem` return this as `std::string`.
With this we can don't need `NetSendCmdString` here and if we later get a real console don't need update this method. 😉